tests/package/packagers_testers/python_standard_library_packagers_testers.py

```diff
@@ -47,17 +49,12 @@
 # ----------------------------------------------------------------------------------------------------------------------
 
 
-NoneType = type(None)  # TODO: Replace with types.NoneType from python 3.10
-
-
-def pack_none() -> NoneType:
+def pack_none() -> None:
     return None
 
 
-def validate_none(result: NoneType) -> bool:
-    # TODO: None values should not be casted to strings when casted to results, once it is implemented in
-    #       'execution._cast_result`, change this validation to `return result is None`.
-    return result == "None"
```

Comment:
Revert it as the cast result change will break stuff when users will manually log None or other non-serializable objects.

Reply:
Not sure I understand you here - this is just the test. Do you refer to the changes in `_cast_result`?
And `None` is JSON serializable:
```py
import json

json.dumps(None) == "null"
```

Reply:
I guess you can test what happen if you pass:
```python
def some_handler(a):
    print(type(a))
    print(a)


my_mlrun_func.run(
    handler="some_handler",
    params={"a": None}
)
```
How will MLRun parse it? give it a try. If you get an actual `None` or `"None"`